src/main/java/net/education/kyivstar/config/MariaDbDeployment.java

```diff
@@ -0,0 +1,37 @@
+package net.education.kyivstar.config;
+
+import ch.vorburger.mariadb4j.DB;
+import ch.vorburger.mariadb4j.DBConfigurationBuilder;
+
+public class MariaDbDeployment {
+
+    private static ConfigDataBase conf;
+
+    public MariaDbDeployment(ConfigDataBase conf) {
+        this.conf = conf;
+    }
+
+    private static DB db = null;
+
+    public static void startEmbeddedMariaDB() {
+        try {
+            DBConfigurationBuilder config = DBConfigurationBuilder.newBuilder();
+            config.setPort(conf.getPort());
+            config.setDataDir(conf.getDirectory());
+            db = DB.newEmbeddedDB(config.build());
+            db.start();
+        } catch (Exception e) {
+            e.printStackTrace();
```

Comment:
Remove the direct printing of the stack trace and replaced it with an exception.